app/screens/home/channel_list/categories_list/categories/categories.tsx

```diff
@@ -29,15 +30,22 @@ const styles = StyleSheet.create({
         marginLeft: -18,
         marginRight: -20,
     },
+    loadingView: {
+        alignItems: 'center',
```

Comment:
you need to justifyContent: 'center' too

Reply:
Maybe I am missing something, but I don't see any material difference between adding this or not (or the previous comment). Why would it be needed?

Reply:
this is how I see it on iOS

![image](https://user-images.githubusercontent.com/6757047/167678266-e8160665-10e5-44c4-afa3-70b6dbf3b097.png)